mlrun/utils/helpers.py

```diff
@@ -267,14 +267,24 @@ def validate_artifact_key_name(
     )
 
 
-def validate_inline_artifact_body_size(body: typing.Union[str, bytes, None]) -> None:
+def validate_artifact_body_size(body: typing.Union[str, bytes, None]) -> None:
+    """
+    Validates the size of the artifact body.
+
+    Args:
+        body: The artifact body, which can be a string, bytes, or None.
+
+    Raises:
+        mlrun.errors.MLRunBadRequestError: If the body exceeds the maximum allowed size.
+    """
```

Comment:
Please follow the same format as here https://github.com/mlrun/mlrun/blob/aee322fc108f2e0a899a7be04a0f0d173c548115/mlrun/utils/helpers.py#L1484-L1488

Reply:
thanks